Calling getChangesetMeta() does not have author's email and name parsed

armenzg opened the pull request:
We were not initalizing a changeset's metadata the same way as when fetched
as part of a call to jsonPushes.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mozilla/firefox-code-coverage-frontend/173)
<!-- Reviewable:end -->

jankeromnes commented on src/containers/diffViewer.jsx:
```diff
@@ -43,7 +43,7 @@ export default class DiffViewerContainer extends Component {
 
   async fetchSetChangesetMeta(node) {
-    const changeset = await (await getChangesetMeta(node)).json();
+    const changeset = await getChangesetMeta(node);
```
Nit: I feel two `await`s on the same line hard to read. Would you mind splitting this line in two, by using an intermediate variable?

armenzg commented:
I've addressed all nits.

I was not thrilled having to add "author" to the signature since I knew the data is in the object but it depends on the fetching method (it's either the `user` or `author` property).
